sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/CreateModelsOptions.cs

```diff
@@ -7,16 +7,22 @@ namespace Azure.DigitalTwins.Core
 {
     /// <inheritdoc />
     [CodeGenModel("DigitalTwinModelsAddOptions")]
-    public partial class CreateModelsOptions
+    internal partial class CreateModelsOptions
     {
-        // This class declaration changes the namespace; do not remove.
+        // This class declaration changes the namespace, class name and property visibility; do not remove.
+
+        // This class contains two properties (TraceParent ,TraceState) that are not intended to be used by the Track 2 SDKs.
+        // Marking these properties as internal.
+        #region internalProperties
```

Comment:
We don't need regions to separate out properties by visibility spec, do we?

Reply:
no worries ! I can certainly remove the regions ... I don't feel that strongly about them.